Sources/IRGen/Function/IULIAContractInitializer.swift

```diff
@@ -22,14 +22,13 @@ struct IULIAContractInitializer {
 
   var isContractFunction = false
 
-  var functionContext: FunctionContext {
-    return FunctionContext(environment: environment, scopeContext: scopeContext, enclosingTypeName: typeIdentifier.name, isInStructFunction: !isContractFunction)
-  }
-
   var parameterNames: [String] {
```

Comment:
A map over explicitParameters would seem to be a clearer way of doing this, but it's OK as it is.

Reply:
I agree, it is nicer that way. So changed.